package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "node_modules/.bin/mocha --require @babel/register test/.setup.js --recursive test",
-    "test:all": "run-s compile dist test test-dist test-lib",
+    "test:all": "run-s build test test-dist test-lib",
```

Comment:
@maoznir  why is that needed as part of this PR?

Reply:
@asisayag2 caught this when test:all failed, compile doesn't exist anymore.